haproxy/values.yaml

```diff
@@ -21,14 +21,23 @@ ingress:
     enabled: true
     secretName: # tls-haproxy
 
+## Extra sidecar containers to add to the deployment
+sidecarContainers: []
+
+
 service:
   port: 30636
   type: ClusterIP
+  ## Extra ports to add to the service (e.g. for sidecar containers)
+  additionalPorts: []
 
 haproxy:
   config: ldap-tls
   frontendPort: 30636
 
+  check:
+    existingSecret: "" # Use existing secret with authentication credentials for checks. Must contain `auth-credentials`
```

Comment:
```suggestion
    existingSecret: "" # Use existing secret with authentication credentials for checks
    existingSecretKey: auth-
    credentials
```

Reply:
I implemented it a little bit different:

```
  check:
    existingSecret: 
      name: "" # Use existing secret with authentication credentials for checks.
      key: "auth-credentials" 
```